CLOUDP-128166: Add "policy" flag to the command (#1370)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-backups-schedule-update.txt

.. _atlas-backups-schedule-update:

=============================
atlas backups schedule update
=============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Update a snapshot backup policies for a cluster.

Reviewer: ```suggestion
Update a snapshot backup policy for a cluster.
```

Author: @sarahsimpers You can upgrade multiple policies by using the flag `--policy` many times. Should it stay as **policies**?

Reviewer: @Razeer123 Thank you, in that case please just remove the "a" so that it reads "Update snapshot backup policies..."

Author: Done!

   * - --clusterName
     - string
     - true
     - Name of the cluster.
   * - --exportBucketId
     - string
     - false
     - Unique identifier that Atlas assigns to the bucket.
   * - --exportFrequencyType
     - string
     - false
     - Frequency associated with the export policy. Value can be daily, weekly, or monthly.
   * - -h, --help
     - 
     - false
     - help for update
   * - --noAutoExport
     - 
     - false
     - Flag that disables automatic export of cloud backup snapshots to the AWS bucket.
   * - --noUpdateSnapshots
     - 
     - false
     - Flag that disables applying the retention changes in the updated backup policy to snapshots that Atlas took previously.
   * - --noUseOrgAndGroupNamesInExportPrefix
     - 
     - false
     - Flag that disables usage of organization and project names instead of organization and project UUIDs in the path for the metadata files that Atlas uploads to your S3 bucket after it finishes exporting the snapshots.
   * - -o, --output
     - string
     - false
     - Output format. Valid values are json, json-path, go-template, or go-template-file.
   * - --policy
     - stringArray
     - false

>      - Array containing a document for each backup policy item in the desired updated backup policy. You must specify it in a format: '--policy policyID,policyItemID,frequencyType,frequencyIntervalNumber,retentionUnit,retentionValue'.

Reviewer: ```suggestion
     - Array containing a document for each backup policy item in the desired updated backup policy. You must specify it in the following format: '--policy policyID,policyItemID,frequencyType,frequencyIntervalNumber,retentionUnit,retentionValue'.
```

   * - --referenceMinuteOfHour
     - int
     - false
     - Minute of the hour to schedule snapshots. Accepted values are between 0 and 59 inclusive.
   * - --restoreWindowDays
     - int
     - false
     - Number of days back in time you can restore to with Continuous Cloud Backup accuracy. Must be a positive, non-zero integer. Applies to continuous cloud backups only.
   * - --updateSnapshots
     - 
     - false
     - Flag that enables applying the retention changes in the updated backup policy to snapshots that Atlas took previously.
   * - --useOrgAndGroupNamesInExportPrefix
     - 
     - false
     - Flag that enables usage of organization and project names instead of organization and project UUIDs in the path for the metadata files that Atlas uploads to your S3 bucket after it finishes exporting the snapshots.

Inherited Options
-----------------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -P, --profile
     - string
     - false
     - Profile to use from your configuration file.

Examples
--------

.. code-block::

>    The following updates a snapshot backup policies for a cluster Cluster0:

Reviewer: ```suggestion
   Update a snapshot backup policy for a cluster named Cluster0:
```
